pages/quiz/quiz.js

```diff
@@ -4,8 +4,8 @@ import { collection, addDoc, getDocs } from 'firebase/firestore';
 import { getAuth } from 'firebase/auth'
 
 const levels = [
-  { number: 1, minScoreToUnlock: 0 },
-  { number: 2, minScoreToUnlock: 2 },
+  { number: 1, minScoreToUnlock: 0, timeLimit: 60 }, 
```

Comment:
This doesn't account for more than two levels. It should be possible to create any number of levels per quiz, so I think this configuration could come from the database.

Reply:
I created a new collection , 'levels', in the database where the time limit and mark to pass to move on to another level is stored and I then fetched that data in my code so that the quiz game could work with it